conan/tools/cmake/toolchain/blocks.py

```diff
@@ -922,12 +922,10 @@ def _is_apple_cross_building(self):
 
     def _get_darwin_version(self, system_name, os_version):
         darwin_versions = {
-            "macos_old": {
+            "Darwin": {
```

Comment:
if we are converting to `Darwin` versions from `Macos`, the key here should be `Macos` not `Darwin`